src/gevent/_hub_primitives.py

```diff
@@ -101,30 +101,27 @@ def __init__(self, objects, hub, timeout, count):
         self._hub = hub
         self._waiter = MultipleWaiter(hub) # pylint:disable=undefined-variable
         self._switch = self._waiter.switch
-        self._timeout = timeout
+        self._timer = (
```

Comment:
Please return this to the old form. If this object is never iterated, we don't want to allocate a native timer.

Reply:
Good point.